giles-eco/src/main/java/edu/asu/diging/gilesecosystem/web/aspects/access/RestSecurityAspect.java

```diff
@@ -77,6 +78,9 @@
     
     @Autowired
     private List<IChecker> checkers;
+
+    @Autowired
+    private GilesTokenConfig tokenConfig;
```

Comment:
also, GilesTokenConfig needs a better name if it does not only contain configurations for tokens